library-app/src/components/Books/BooksList/ManageBookForm.module.css

```diff
@@ -0,0 +1,44 @@
+.form-wrapper {
+  display: flex;
+  flex-direction: column;
+  /* gap: 2rem; */
+  height: 100%;
+  justify-content: flex-start;
+  height: 100%;
```

Comment:
You have height:100% two times in the same block of properties, please remove one of them